src/tests/ov_helpers/ov_snippets_models/src/subgraph_group_normalization.cpp

```diff
@@ -0,0 +1,99 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "subgraph_group_normalization.hpp"
+#include <snippets/op/subgraph.hpp>
+
+namespace ov {
+namespace test {
+namespace snippets {
+
+std::shared_ptr<ov::Model> GroupNormalizationFunction::initOriginal() const {
+    auto data = std::make_shared<op::v0::Parameter>(precision, input_shapes[0]);
+    auto scale = std::make_shared<op::v0::Parameter>(precision, input_shapes[1]);
+    auto shift = std::make_shared<op::v0::Parameter>(precision, input_shapes[2]);
+    const auto groupNormalization = std::make_shared<ov::op::v12::GroupNormalization>(data, scale, shift, num_groups, epsilon);
+    return std::make_shared<ov::Model>(NodeVector{groupNormalization}, ParameterVector{data, scale, shift});
+}
+
+std::shared_ptr<ov::Model> GroupNormalizationFunction::initReference() const {
+    auto data = std::make_shared<op::v0::Parameter>(precision, input_shapes[0]);
+    auto scale = std::make_shared<op::v0::Parameter>(precision, input_shapes[1]);
+    auto shift = std::make_shared<op::v0::Parameter>(precision, input_shapes[2]);
+    auto data_ = std::make_shared<op::v0::Parameter>(precision, data->get_shape());
+    auto scale_ = std::make_shared<op::v0::Parameter>(precision, scale->get_shape());
+    auto shift_ = std::make_shared<op::v0::Parameter>(precision, shift->get_shape());
```

Comment:
Let's use `get_output_partial_shape(0)` or `input_shapes[i]` instead of `get_shape` to avoid failures in case of dynamic shapes

Reply:
updated! thanks Vladislav!